src/components/webview/MainProvider.ts

```diff
@@ -241,6 +241,13 @@ export class MainViewProvider implements WebviewViewProvider {
 					message.searchPhrase,
 				]),
 			);
+
+			this.__store.dispatch(
```

Comment:
why is this action needed?

Reply:
to recalculate explorerNodes. i've tried to put search logic to the selector where it should be, but had a problem that i cannot keep correct tree structure this way. so i had to put this logic to reducer.